pkg/controller/factory.go

```diff
@@ -126,8 +132,16 @@ func (f *GardenControllerFactory) Run(stopCh <-chan struct{}) {
 
 	// Shutdown handling
 	<-stopCh
-	logger.Logger.Info("I have received a stop signal and will no longer watch events of the Garden API group.")
-	logger.Logger.Infof("Number of remaining workers -- Shoot: %d, Seed: %d, Quota: %d, CloudProfile: %d, SecretBinding: %d, BackupInfrastructure: %d", shootController.RunningWorkers(), seedController.RunningWorkers(), quotaController.RunningWorkers(), cloudProfileController.RunningWorkers(), secretBindingController.RunningWorkers(), backupInfrastructureController.RunningWorkers())
+	logger.Logger.Infof("I have received a stop signal and will no longer watch events of the Garden API group.")
+	logger.Logger.Infof("== NUMBER OF REMAINING WORKERS ==")
```

Comment:
Can't https://golang.org/pkg/text/tabwriter/ be used for this?

Reply:
Done.